openshift-install-powervs

```diff
@@ -13,7 +13,7 @@ limitations under the License.
 '
 #-------------------------------------------------------------------------
 set -e
-VERSION="v1.0.3"
+VERSION="v1.0.4"
```

Comment:
Not intended?

Reply:
I did not make this change, It comes from the merged from upstream.

Reply:
May be it was added while resolving conflicts during merge/rebase.

Reply:
I have checked the `VERSION` on devel branch at github, it is 1.0.4. If it is OK, I'll push a new commit with signed-off-by.

Reply:
I have synced my devel branch with upstream devel branch, but the version still show changed. Could be the code in github need to be synced before I push my code? How to sync my fork to origin?